drivers/vmwarevsphere/vsphere.go

```diff
@@ -72,15 +72,14 @@ type Driver struct {
 	CloneFrom              string
 	SSHPassword            string
 	SSHUserGroup           string
-
-	vms          map[string]*object.VirtualMachine
-	soap         *govmomi.Client
-	ctx          context.Context
-	finder       *find.Finder
-	datacenter   *object.Datacenter
-	networks     map[string]object.NetworkReference
-	hostsystem   *object.HostSystem
-	resourcepool *object.ResourcePool
+	vms                    map[string]*object.VirtualMachine
```

Comment:
this is just white space changes it looks like.. maybe unecessary?

Reply:
fixed w/ go fmt, file watcher was busted